go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/stretchr/testify v1.4.0
 	golang.org/x/net v0.0.0-20190923162816-aa69164e4478 // indirect
 	golang.org/x/text v0.3.2 // indirect
-	google.golang.org/grpc v1.23.1
+	google.golang.org/grpc v1.27.1
```

Comment:
protobuf failed for me before I upgraded grpc.

Reply:
What do you mean failed? `make protobuf`?

Reply:
`make protobuf` produced a pb.go file that didn't compile.

Reply:
That's very strange. Maybe you have a too new version of protoc-gen-go? You want to use same version of protoc-gen-go as we use for github.com/golang/protobuf (v1.3.2) which is the same version as hashicorp/go-plugin uses in v1.0.1. Related issue #71

I see now that there's finally been some updates to go-plugin and they've tagged a version [1.1.0](https://github.com/hashicorp/go-plugin/releases) which includes same gRPC version as you're using here. 

May I suggest create an issue for upgrading to gp-plugin v1.1.0 so we can fix that in a separate PR after v6.7 beta?